### ISSUE TITLE
Role=status property is missing for Status related field in Admin screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#874]
**Describe the bug**
role=status property is missing for all status related fields.

**To Reproduce**

1. Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test10&path=%2FSites%2Ftest10%2F&pathType=page)
2. Click plus icon given against "Assigned" label in the left side of screen.
3. Select sites from the  pop up, and then click save. 
4. Status sections are displayed in sequence, after save button clicked.
5. First is the status popup and second below the Assigned section, there is another section where status is displayed, like "Loading, please wait...".
6. In these two sections, the Role=status property is missing 

**Expected behavior**
role=status property must be available for all status related fields.

**Additional context**
Navigate all the options available for similar issue and fix.
